test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -104,9 +107,21 @@ class AndroidArgs(
             "Option num-uniform-shards cannot be specified along with max-test-shards. Use only one of them"
         )
 
+        if (!(isRoboTest xor isInstrumentationTest)) throw FlankFatalError(
+            "Option test xor robo-directives xor robo-script must be specified"
```

Comment:
How about `xor` -> `or`?
Maybe we could use sth like `One of bla bla bla`. Just thinking

Reply:
Flank is for programmers not end users, I guess every programmer knows what `xor` does mean. Anyway if you propose something better than `One of bla bla bla` ;), I can replace.

Reply:
we have a community of QA engineers using Firebase Test Lab, they may not be familiar with what xor means 🙂